unica: mods: bootanim: determine device resolution from BACK_CAMERA_R…

mattchengg opened the pull request:
I can't think of a better way

mattchengg committed:
unica: mods: bootanim: determine device resolution from BACK_CAMERA_RESOLUTION_FULL_RATIO

Updated boot animation script to read camera resolution from XML and determine appropriate animation blobs based on aspect ratio.

majaahh reviewed (commented):
You should use ```EVAL``` for ```cp``` commands and ```if``` instead of ```case``` to match the build system and other scripts code style

mattchengg committed:
unica: mods: bootanim: Refactor boot animation resolution detection

- Replace case statement with if-elif
- Use EVAL wrapper for cp commands
- Define aspect ratios with delta for clearer range checking

mattchengg committed:
unica: mods: bootanim: Unset variables

majaahh commented on unica/mods/bootanim/customize.sh:
```diff
@@ -16,19 +16,21 @@ read -r WIDTH HEIGHT RATIO <<< "$(awk '
+    
+    if [ "$(echo "$RATIO >= $RATIO_2340 - $RATIO_2340_DELTA && $RATIO <= $RATIO_2340 + $RATIO_2340_DELTA" | bc -l)" = "1" ]; then
+        LOG "- Adding 2024 boot animation blobs (1080x2340)"
+        EVAL "cp -a '$MODPATH/1080x2340/'* '$WORK_DIR/system/system/media'" || return 1
```
you should use ```\"$MODPATH/res/\"*```. Apply this for other commands too

mattchengg committed:
use \"$MODPATH/res/\"*

mattchengg committed:
Merge branch 'salvogiangri:sixteen' into bootanim

salvogiangri commented:
Merged with 7930b7afb785061e3730aba8003850ddafb05c8d.
